aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -405,6 +406,7 @@ impl From<FeatureFlag> for AptosFeatureFlag {
             FeatureFlag::EnableCaptureOption => AptosFeatureFlag::ENABLE_CAPTURE_OPTION,
             FeatureFlag::EnableTrustedCode => AptosFeatureFlag::ENABLE_TRUSTED_CODE,
             FeatureFlag::EnableEnumOption => AptosFeatureFlag::ENABLE_ENUM_OPTION,
+            FeatureFlag::EnableLayoutCaches => AptosFeatureFlag::ENABLE_LAYOUT_CACHES,
```

Comment:
this cache doesn't change any semantics, so use a local config is probably safer and easier than the feature flag. similar to paranoid mode